src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/detection_output_gpu_ref.cl

```diff
@@ -415,6 +411,10 @@ KERNEL (detection_output_stage_1_sort_caffe)(__global uchar *buffer0,
     const int localClassId = get_local_id(1);
     __local int __range[LOCAL_CLASS_NUM][LOCAL_WORK_NUM * 2];
 
+    // Create an auxiliary stack for QuickSort
+    const int kStackSize = 100;
+    __local int stack[kStackSize];
```

Comment:
This is being shared by local work items. So it should be kStaticSize * LWS and each WI should use its own space.. ? 
Did you check the accuracy with this change?

Reply:
Thanks, you are right, I updated DO_STAGE_1_CAFFE and DO_STAGE_3_CAFFE with respect to maximum of available SLM memory
The accuracy looks good